dev/com.ibm.ws.jsp/bnd.bnd

```diff
@@ -61,7 +61,8 @@ Export-Package: com.ibm.ws.jsp.runtime;version=2.2;thread-context=true, \
    com.ibm.ws.jsp.webcontainerext;version=2.2, \
    com.ibm.ws.jsp.webcontainerext.ws;version=2.2, \
    org.apache.jasper.compiler;thread-context=true;version=2.2, \
-   org.apache.jasper.runtime;thread-context=true;version=2.2
+   org.apache.jasper.runtime;thread-context=true;version=2.2, \
+   com.ibm.wsspi.el
```

Comment:
Mostly used internally within com.ibm.ws.jsp.

Otherwise, this package is exported to build the com.ibm.ws.cdi.web bundle. 

https://github.com/OpenLiberty/open-liberty/blob/2fd4a880754c37a988c5ed9ac4f1ea5988e465d6/dev/com.ibm.ws.cdi.web/src/com/ibm/ws/cdi/web/impl/el/WrappedELExpressionFactory.java#L28